bin/cli.js

```diff
@@ -65,18 +63,13 @@ const cliFlags = yargs
     return true;
   }).argv;
 
-//Get all explicitly terminal set options, does not include url because url has no default option
```

Comment:
honestly i couldn't figure out why we need this. :)

(i'm mostly going off how the lighthouse CLI works and it's not needed such behavior)

pedro, let me know if i'm missing something.